user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,461 @@
+
+This capability is usually used by the system administrators.
+
+
+.. _admin-capabilities-mailchimp:  
+
+MailChimp manipulations
+-----------------------
+
+The capability is meaningful if the system has been integrated with Mailchimp. Such integration enables users to utilize 
+MailChimp as an email campaign engine  as described in the guide on 
+:ref:`Integration with MailChimp <user-guide-mc-integration>`. 
+
+When using the functionality, the user can synchronize lists of contacts in OroCRM and MailChimp, use them to create 
+email campaigns in MailChimp and import them to OroCRM – the MailChimp campaign statistics can then be used to analyze 
+the campaign efficiency. All of these operations on the OroCRM side will only be available if the capability has been 
+set to  *"System"*.
+
+The capability is usually enabled for marketing associates and other managers responsible for email campaigns.
+
+
+.. _admin-capabilities-config-entities:  
+
+Manage configurable entities
+----------------------------
+
+Many entities in OroCRM can be configured from the UI, as described in the 
+:ref:`Entities guide <user-guide-entity-management-from-UI>`. So, the system administrator can change the attachments 
+settings, define whether the entity should be displayed on a grid and/or a View page, whether it will be exported 
+to a .CSV file, and define other settings. For some of them it is also possible to add new fields, as described in the 
+:ref:`Entity Fields guide <user-guide-field-management>`. 
+
+These actions are only available if this capability is set to “System”, and are usually performed by the system 
+administrators.
+
+
+.. _admin-capabilities-org-calendar-events: 
+
+Manage organization calendar events
+-----------------------------------
+
+The capability enables users to create, edit and delete events in the organization-wide calendars, which are described 
+in more details in the :ref:`corresponding section <user-guide-calendars-system>` of the *Calendars Overview* guide.
+
+This capability can be utilized by the organization-level managers and HRs or other employees authorized to manage the 
+organization-wide calendars
+
+Users can view organization-wide calendars, add them to their own calendar views and copy related events to their own 
+calendars, regardless of this capability.
+
+
+.. _admin-capabilities-sys-calendar-events: 
+
+Manage system calendar events
+-----------------------------
+
+The capability enables users to create, edit and delete events in the system-wide calendars, which are described 
+in more details in the :ref:`corresponding section <user-guide-calendars-system>` of the *Calendars Overview* guide.
+
+This capability can be utilized by the company managers and HRs or other employees authorized to manage calendars at the 
+system-wide level.
+
+Users can view organization-wide calendars, add them to their own calendar views and copy related events to their own 
+calendars, regardless of this capability.
```

Comment:
Comment: Once again, please see my comment above regarding the "regardless of this capability" clause.